learn-express/routes.js

```diff
@@ -0,0 +1,66 @@
+const express = require("express")
+const router = express.Router()
+
+// import our model
+const Post = require("./models/Post")
+
+// Get all posts
+router.get("/posts", async (req, res) => {
+	const posts = await Post.find()
```

Comment:
Don't mix logic with routes you should have routes and controllers that will have a callback function or class